openshift/4.0/scripts/oc-sonar-scan.sh

```diff
@@ -1,68 +0,0 @@
-#!/bin/bash
-#
```

Comment:
This wasn't used anywhere in the repo.

Runs a Sonar scan for a JavaScript/TypeScript codebase, including unit tests and coverage.